staging/src/k8s.io/component-base/metrics/processstarttime_others.go

```diff
@@ -0,0 +1,38 @@
+// +build !windows
+
```

Comment:
why not use file staging/src/k8s.io/component-base/metrics/processstarttime.go ?

Reply:
The function `getProcessStart` needs to be defined uniquely across windows and other os's or you get a build error:

```
vendor/k8s.io/component-base/metrics/processstarttime_windows.go:25:6: getProcessStart redeclared in this block
```

This is a common pattern in other areas: https://github.com/kubernetes/kubernetes/blob/ba5f5bea64a7b42265e581e8c7fe633336bec79a/pkg/kubelet/kuberuntime/security_context_others.go#L30